src/plugins/dashboard/public/application/lib/build_dashboard_container.ts

```diff
@@ -122,7 +122,6 @@ export const buildDashboardContainer = async ({
         gridData: originalPanelState.gridData,
         type: incomingEmbeddable.type,
         explicitInput: {
-          ...originalPanelState.explicitInput,
```

Comment:
ℹ️ Not needed, allows a TSVB library panel to be replaced by a Lens by value panel